pkg/controller/container-runtime-config/helpers.go

```diff
@@ -144,9 +144,9 @@ func findStorageConfig(mc *mcfgv1.MachineConfig) (*igntypes.File, error) {
 }
 
 func findRegistriesConfig(mc *mcfgv1.MachineConfig) (*igntypes.File, error) {
-	ignCfg, report, err := ign.Parse(mc.Spec.Config.Raw)
+	ignCfg, err := ctrlcommon.ParseAndConvertConfig(mc.Spec.Config.Raw)
 	if err != nil {
-		return nil, fmt.Errorf("parsing Registries Ignition config failed with error: %v\nReport: %v", err, report)
```

Comment:
Just to confirm (bc i can't follow the new github diffs very well) : does this new returned error still return a report?

Reply:
Not 100% sure on the reports but from what I recall they were mostly empty and the error message captured the reason

Reply:
On actually the report is now captured via the error message that the parse function gives

Reply:
ok perfect!

Reply:
wait.. which parse function?

Reply:
ok both IgnParseWrapper and ParseAndConvertConfig wrap the report in the error! :+1: